README.md

```diff
@@ -31,49 +33,38 @@ cd your-repo
 ### 2. Install Dependencies
 
 ```sh
-cd backend
-npm install
-
-cd ../frontend
 npm install
 ```
 
-### 3. Install Dependencies
-
-Set Up PostgreSQL with Docker (docker-compose.yml provided)
+### 4. Set Environment Variables
 
-### 4. Environment Variables
-
-in backend root:
+In the `backend` directory:
 
 ```sh
-BLOCKSTREAM_API_URL=https://blockstream.info/testnet/api #test-api
-DATABASE_URL=postgres://<<your-user>>>>:<<your-passward>>>>@localhost:5432/<<bitsnark or other name>>
-
+BLOCKSTREAM_API_URL=https://blockstream.info/testnet/api
+DATABASE_URL=postgres://<user>:<password>@localhost:5432/bitsnark
 ```
 
 ### 5. Start the PostgreSQL Container
 
-Run the following command to start the PostgreSQL container:
+Still in the `backend` directory:
 
 ```sh
 docker-compose up -d
 ```
 
-### 5.Run the Backend and Frontend
```

Comment:
I think it's better to split this into two sections.

Reply:
ok